src/client/a2/main/grid/layers/filters/radar-filter.ts

```diff
@@ -0,0 +1,88 @@
+import * as PIXI from 'pixi.js'
+
+export class RadarFilter extends PIXI.Filter {
+  constructor(private maxTime: number) {
+    super(
```

Comment:
I think webpack should be able to deal with this in a straightforward way, and inline file contents so you don't actually need to deal with the assets directory.